src/vm/dllimport.cpp

```diff
@@ -6078,9 +6096,13 @@ static void DetermineLibNameVariations(const char* const** libNameVariations, in
             static const char* const SuffixFirst[] =
             {
                 "%.0s%s%s", // name+suffix
+#ifdef FEATURE_PAL
```

Comment:
Similar for names that end with a `.`. We should not do any probing for them on Windows - to maintain the behavior documented for them on MSDN.